sites/partners/src/components/listings/PaperListingForm/index.tsx

```diff
@@ -343,6 +348,10 @@ const ListingForm = ({ listing, editMode, setListingName, updateListing }: Listi
             formData.listingSection8Acceptance = YesNoEnum.no
           }
 
+          if (!enableNonRegulatedListings) {
+            formData.listingType = null
```

Comment:
issue: this is the line that is causing the issue. it should be "undefined" instead of "null". Prisma treats undefined as it should do nothing whereas "null" tries to save null